src/stylesheets/components/_sidenav.scss

```diff
@@ -1,8 +1,28 @@
 
 .usa-sidenav-list {
   @include usa-sidenav-list;
+  @include margin(null (-$site-margins-mobile) null (-$site-margins-mobile));
+  border-bottom: 1px solid;
+  border-top: 1px solid;
+
+  @include media($medium-screen) {
+    @include margin(null 0);
+  }
+
+  @include media($large-screen) {
+    border: none;
+    margin: 0;
```

Comment:
When I removed this code and it didn't have an effect on the margins because the top and bottom margins are already removed in the `$medium-screen` media query above. Can this be removed?

Reply:
will do